internal/fs/fs_test.go

```diff
@@ -141,7 +144,7 @@ func (t *fsTest) SetUpTestSuite() {
 		appendThreshold: 0,
 		tmpObjectPrefix: ".gcsfuse_tmp/",
 	}
-	t.serverCfg.RenameDirLimit = 5
+	t.serverCfg.RenameDirLimit = RenameDirLimit
 	t.serverCfg.SequentialReadSizeMb = 200
```

Comment:
Can we create constant for SequentialReadSizeMb as well ?

Reply:
done.